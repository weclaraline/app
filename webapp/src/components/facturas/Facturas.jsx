import { Grid, makeStyles, Paper, Typography } from "@material-ui/core";
import React from "react";
import ConceptoDeduciblesComponent from "../facturas/ConceptoDeduciblesComponent";
import FAQEnlacesComponent from "../facturas/FAQEnlacesComponent";

const useStyles = makeStyles( (theme) => {
    const { main } = theme.palette.primary;
    const { overline, h1 } = theme.typography;
    return (
        {
            headerRow: {
                height: "200px",
                background: main,
                display: "flex",
                alignItems: "center",
                justifyContent: "center"
            },
            paper: {
                alignContent: "center",
                justifyContent: "center",
                color: "white",
                background: "transparent"
            },
            header: {
                h1
            },
            overline: {
                overline,
            },
        }
    )
});

const Facturas = () => {

    const classes = useStyles();

    return (
        <Grid container spacing={3}>
            <Grid 
                data-testid="title_banner" 
                item 
                className={classes.headerRow}
                xs={12}
            >
                <Paper elevation={0} className={classes.paper}>
                    <Typography variant={"h4"}>
                        Guía para solicitar facturas.
                    </Typography>
                    <Typography variant={"h6"}>
                        Sigue los pasos para que fácilmente...
                    </Typography>
                </Paper>
            </Grid>
            <Grid item xs={8}>
                <ConceptoDeduciblesComponent />
            </Grid>
            <Grid item xs={4}>
                <FAQEnlacesComponent />
            </Grid>    
        </Grid>
    );
}

export default Facturas;