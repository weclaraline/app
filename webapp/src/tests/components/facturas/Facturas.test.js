import '../../testing_utils';
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Facturas from "../../../components/facturas/Facturas";

it("Renders original content", ()=> {
    const tree = shallow(<Facturas />);
    expect(toJson(tree)).toMatchSnapshot();
});