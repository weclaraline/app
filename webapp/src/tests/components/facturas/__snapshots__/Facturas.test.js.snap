// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders original content 1`] = `
<div>
  PÃ¡gina de Facturas.
</div>
`;
